src/app/styl/views/main_window.styl

```diff
@@ -1,8 +1,16 @@
 html,
 body,
+    text-rendering: optimizelegibility
```

Comment:
this actually won't work. you're removing height/cursor/user-select/bg-color from body